### PR TITLE
Noise model inserter

### PR DESCRIPTION
**Pull request by gadial**
### Summary
A utility addressing issue #212 : Inserting to a circuit Kraus gates representing the noises from a given noise model.

### CODE REVIEW

**Comment by gadial** [comment]
Quick update - testing cannot proceed due to a blocker bug in Terra, https://github.com/Qiskit/qiskit-terra/issues/2751.

File: qiskit/providers/aer/noise/utils/noise_model_inserter.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+"""
+Noise model inserter module
+The goal of this module is to add QuantumError gates (Kraus gates) to a circuit
+based on a given noise model. The resulting circuit cannot be ran on a quantum computer
+but can be handled correctly by simulators
+"""
+import qiskit.compiler
+
+
+def add_errors(circuit, noise_model, transpile=False):
```
Comment (chriseclectic) [inline qiskit/providers/aer/noise/utils/noise_model_inserter.py]: Can you rename this to `insert_noise` and make it so it works with either a single input circuit, or a list of input circuits similarly to`transpile/execute/assemble`

File: qiskit/providers/aer/noise/utils/noise_model_inserter.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        Returns:
+            QuantumCircuit: The new circuit with the added Kraus gates
+        """
+    errors = noise_model._default_quantum_errors.items()
+    error_dict = {name: qubit_dict for (name, qubit_dict) in errors}
+    if transpile:
+        transpiled_circuit = qiskit.compiler.transpile(circuit, basis_gates=noise_model.basis_gates)
+    else:
+        transpiled_circuit = circuit
+    result_circuit = circuit.copy(name=transpiled_circuit.name + '_with_errors')
```
Comment (chriseclectic) [inline qiskit/providers/aer/noise/utils/noise_model_inserter.py]: similarly `_with_noise` here

File: qiskit/providers/aer/noise/utils/noise_model_inserter.py (hunk 1 above)
Comment (gadial) [reply qiskit/providers/aer/noise/utils/noise_model_inserter.py]: Addressed in 72210203ad4bd445cc05a9c505b7328987f0a399

**Pull request merged**